WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -167,7 +167,7 @@ - (NSDictionary *)fb_accessibilityTree
   return [self.class accessibilityInfoForElement:snapshot];
 }
 
-+ (NSDictionary *)dictionaryForElement:(id<FBXCElementSnapshot>)snapshot recursive:(BOOL)recursive
++ (NSDictionary *)dictionaryForElement:(id<FBXCElementSnapshot>)snapshot recursive:(BOOL)recursive excludedAttributes:(NSArray *) excludedAttributes
```

Comment:
Please format long strings, so different args are aligned to new lines

Reply:
Done